dispike/incoming/discord_types/member.py

```diff
@@ -30,9 +30,9 @@ class Config:
     permissions: str
     pending: bool
     nick: typing.Union[None, str]
-    mute: bool
+    mute: bool = None
```

Comment:
If this is optional -- do ``typing.Optional[bool]``

Reply:
Nope, it's not given for some reason in a user context interaction.